RunETL/runETL.cmd

```diff
@@ -1,66 +1,71 @@
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'date_last_export', '%date_last_export%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'input_filename', '%input_filename%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'diagnosis_filename', '%diagnosis_filename%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'drug_filename', '%drug_filename%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'tissue_filename', '%tissue_filename%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'other_therapies_filename', '%other_therapies_filename%' | Set-Content docker-compose.yml"
-powershell -Command "(Get-Content docker-compose.yml) -creplace 'esfurn_site', '%esfurn_site%' | Set-Content docker-compose.yml"
-
-docker login harbor.esfurn.org
-docker compose pull
-docker compose run --rm --name etl etl
+::#############
+::## RUN ETL ##
+::#############
+SET REGISTRY_ETL_RUNNER=harbor.honeur.org
+SET REPOSITORY_ETL_RUNNER=library
+SET IMAGE_ETL_RUNNER=etl-runner
+SET VERSION_ETL_RUNNER=1.1.4
+SET TAG_ETL_RUNNER=%VERSION_ETL_RUNNER%
+
+set LOG_FOLDER_HOST=%PWD%/log
+set DATA_FOLDER_HOST=%PWD%/data
+set QA_FOLDER_HOST=%PWD%/qa
+
+echo "Pull ETL runner image"
+
+docker pull %REGISTRY_ETL_RUNNER%/%REPOSITORY_ETL_RUNNER%/%IMAGE_ETL_RUNNER%:%TAG_ETL_RUNNER%
+
+SET REGISTRY=harbor.esfurn.org
+SET REPOSITORY=etl
+SET CDM_SCHEMA=omopcdm
+SET VOCAB_SCHEMA=%CDM_SCHEMA%
+
+echo "Download ETL questions"
+curl -fsSL https://raw.githubusercontent.com/solventrix/ESFURN-Setup/blob/master/RunETL/questions.json --output questions.json
+
+:: Create the env file 
+:: etl image
+echo THERAPEUTIC_AREA=esfurn > etl-runner.env
+echo REGISTRY=%REGISTRY% >> etl-runner.env
+echo ETL_IMAGE_NAME=%REPOSITORY%/esfurn_etl >> etl-runner.env
+echo ETL_IMAGE_TAG=current >> etl-runner.env
+:: logs
+echo LOG_FOLDER_HOST=%LOG_FOLDER_HOST% >> etl-runner.env
+echo LOG_FOLDER=/log >> etl-runner.env
+echo VERBOSITY_LEVEL=%VERBOSITY_LEVEL% >> etl-runner.env
+:: schemas
+echo DB_SCHEMA=%CDM_SCHEMA% >> etl-runner.env
+echo VOCAB_SCHEMA=%VOCAB_SCHEMA% >> etl-runner.env
+:: site
+echo ESFURN_SITE=%ESFURN_SITE% >> etl-runner.env
+:: source data
+echo DATA_FOLDER_HOST=%DATA_FOLDER_HOST% >> etl-runner.env
+echo DATA_FOLDER=/data >> etl-runner.env
+echo INPUT_DATA=%INPUT_DATA% >> etl-runner.env
+echo DIAGNOSIS_CODE_MAPPING=%DIAGNOSIS_CODE_MAPPING% >> etl-runner.env
+echo DRUG_CODE_MAPPING=%DRUG_CODE_MAPPING% >> etl-runner.env
+echo SITE_COLLECTION_MAPPING=%SITE_COLLECTION_MAPPING% >> etl-runner.env
+echo OTHER_THERAPIES_MAPPING=%OTHER_THERAPIES_MAPPING% >> etl-runner.env
+echo LAST_DATA_EXPORT=%LAST_DATA_EXPORT% >> etl-runner.env
```

Comment:
echo >> doesn't work on windows. Add each env variable to the docker run command using
--env